python-clusters/create-gke-cluster/cluster.json

```diff
@@ -79,20 +109,29 @@
         {
             "name": "s-nodes",
             "type":"SEPARATOR",
-            "label": "Cluster nodes"
+            "label": "Cluster nodes",
+            "visibilityCondition": "!model.isAutopilot"
         },
         {
             "name": "nodePools",
             "label": "Node pools",
             "description": "Node pools to create in the cluster",
             "type": "PRESETS",
             "parameterSetId": "node-pool-request",
-            "mandatory": true
+            "mandatory": true,
+            "visibilityCondition": "!model.isAutopilot"
         },
          {
             "type":"SEPARATOR",
             "label": "Advanced options"
         },
+        {
+            "name": "isAutopilot",
```

Comment:
I installed the plugin but I can't find how I can enable this option, @nikopol would you have an idea where this should appear?

Reply:
The first time, I had to restart dss to make it appears.